[2024-10-31T16:09:34.356+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-31T16:09:34.368+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: datapipeline_new.pivot_data_train manual__2024-10-31T16:09:16.507457+00:00 [queued]>
[2024-10-31T16:09:34.373+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: datapipeline_new.pivot_data_train manual__2024-10-31T16:09:16.507457+00:00 [queued]>
[2024-10-31T16:09:34.373+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-10-31T16:09:34.381+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): pivot_data_train> on 2024-10-31 16:09:16.507457+00:00
[2024-10-31T16:09:34.391+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=167) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-31T16:09:34.392+0000] {standard_task_runner.py:63} INFO - Started process 185 to run task
[2024-10-31T16:09:34.392+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'datapipeline_new', 'pivot_data_train', 'manual__2024-10-31T16:09:16.507457+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/dag_script.py', '--cfg-path', '/tmp/tmpgnjxisiy']
[2024-10-31T16:09:34.393+0000] {standard_task_runner.py:91} INFO - Job 109: Subtask pivot_data_train
[2024-10-31T16:09:34.601+0000] {task_command.py:426} INFO - Running <TaskInstance: datapipeline_new.pivot_data_train manual__2024-10-31T16:09:16.507457+00:00 [running]> on host 8b35fed25b46
[2024-10-31T16:09:34.632+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='MLOPS' AIRFLOW_CTX_DAG_ID='datapipeline_new' AIRFLOW_CTX_TASK_ID='pivot_data_train' AIRFLOW_CTX_EXECUTION_DATE='2024-10-31T16:09:16.507457+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-31T16:09:16.507457+00:00'
[2024-10-31T16:09:34.632+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-31T16:09:34.644+0000] {logging_mixin.py:188} INFO - Data loaded from /opt/***/dags/DataPreprocessing/src/data_store_pkl_files/train_data/train_data.pkl
[2024-10-31T16:09:34.698+0000] {logging_mixin.py:188} INFO - Date column converted to datetime.
[2024-10-31T16:09:34.722+0000] {logging_mixin.py:188} INFO - Data pivoted successfully.
[2024-10-31T16:09:34.726+0000] {logging_mixin.py:188} INFO - Pivoted DataFrame saved as '/opt/***/dags/DataPreprocessing/src/data_store_pkl_files/train_data/pivoted_train_data.pkl'.
[2024-10-31T16:09:34.726+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-31T16:09:34.727+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-31T16:09:34.731+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=datapipeline_new, task_id=pivot_data_train, run_id=manual__2024-10-31T16:09:16.507457+00:00, execution_date=20241031T160916, start_date=20241031T160934, end_date=20241031T160934
[2024-10-31T16:09:34.781+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-10-31T16:09:34.909+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-31T16:09:34.910+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
